src/jit/x64/emit.dasc

```diff
@@ -1917,9 +1917,38 @@ static void load_call_arg(MVMThreadContext *tc, MVMJitCompiler *compiler, MVMJit
         | mov TMP6, qword WORK[arg.v.reg];
         | lea TMP6, STOOGE:TMP6->data;
         break;
+    case MVM_JIT_REG_DYNIDX:
+        | get_cur_op TMP5;
+        | xor TMP6, TMP6;
+        | mov TMP6w, word [TMP5 + arg.v.reg*2]
+        | mov TMP6, qword [WORK + TMP6*8];
+        break;
     case MVM_JIT_DATA_LABEL:
         | lea TMP6, [=>(arg.v.lit_i64)];
         break;
+    case MVM_JIT_SAVED_RV:
+        | mov TMP6, qword [rbp-0x28];
+        break;
```

Comment:
There are other ways to do this; specifically, I've been using the args array as a temporary buffer in the compilation of unless_o and if_o. Not sure if that is possible in this case, but it would save you the implementation of SAVED_RV.
Alternatively, you can give  `[rbp-0x28]` a symbolic name with 
`|.define SAVED_RV, [rbp-0x28]`

Reply:
I don't see an advantage of using the args buffer over just putting it on the stack. I'd have to locate the args buffer first, at least for ahead of time compilation causing unnecessary memory churn. I have it the symbolic name. Works like a charm :)